packages/core/src/types/Vector.ts

```diff
@@ -47,6 +47,14 @@ export class Vector2 implements Type {
   public static readonly up = new Vector2(0, 1);
   public static readonly down = new Vector2(0, -1);
```

Comment:
These properties appear to be incorrect, but since they belong to a previous commit, and changing them might break existing projects, I elected to leave them alone and flag them for review.

From my understanding of Motion Canvas, the following changes should be made:
```suggestion
  public static readonly up = new Vector2(0, -1);
  public static readonly down = new Vector2(0, 1);
```

Reply:
They are "correct" in that they're aligned with a mathematical expectation for the `up` and `down` vectors.
Regardless, we cannot change them right now as that would be considered a breaking change. This would require raising the version to `4.x.x` which we are not ready for just yet.

<details>
<summary>Here's a <a href="https://discord.com/channels/1071029581009657896/1175315979174162453">Discord conversation</a> with @hhenrichsen for more detail</summary>

![image](https://github.com/motion-canvas/motion-canvas/assets/64662184/d1eb660c-166f-47eb-a259-1ca4509bd0c1)

</details>